grave-pile.lic

```diff
@@ -0,0 +1,36 @@
+custom_require.call(%w[events common-travel common])
+
+
+junk_list = get_settings.grave_junk.map { |x| /\b#{x}/i }
+
+DRCT.walk_to(10_291)
+loop do
+  break if checkbleeding
+  break if checkhealth < 75
+
+  until checkright || checkleft
+    fput('stand') unless checkstanding
+    if /injured/ =~ DRC.bput('dig pile', 'roundtime', 'You are too injured', 'Perhaps that would work better')
+      DRCT.walk_to(19_104)
+      exit
+    end
+    pause 1
+    waitrt?
+  end
+
+  while thing = DRC.left_hand || DRC.right_hand
+    case thing
+    when *junk_list
+      fput("put #{thing} in bin")
```

Comment:
`bput("put #{thing} in bin", 'You drop')` instead maybe?

Reply:
Does `dispose_trash` do what you need?